lib/api/kuzzle.js

```diff
@@ -33,22 +34,28 @@ const
   FunnelController = require('./controllers/funnelController'),
   HotelClerk = require('./core/hotelClerk'),
   Janitor = require('./core/janitor'),
-  IndexCache = require('./core/indexCache'),
-  InternalEngine = require('../services/internalEngine'),
   Notifier = require('./core/notifier'),
   PassportWrapper = require('./core/auth/passportWrapper'),
   PluginsManager = require('./core/plugins/pluginsManager'),
   Repositories = require('./core/models/repositories'),
   RouterController = require('./controllers/routerController'),
-  Services = require('../services'),
   Statistics = require('./core/statistics'),
   TokenManager = require('./core/auth/tokenManager'),
   Validation = require('./core/validation'),
-  InternalEngineBootstrap = require('../services/internalEngine/bootstrap'),
+  InternalIndexBootstrap = require('./core/storage/bootstrap/internalIndexBootstrap'),
+  IndexStorage = require('./core/storage/indexStorage'),
+  CacheEngine = require('./core/cache/cacheEngine'),
+  StorageEngine = require('./core/storage/storageEngine'),
   Vault = require('./core/vault'),
   runShutdown = require('../util/shutdown'),
   Logger = require('../util/log');
 
+// Add meaningful stacktrace in Node.js 10
+// @todo must be removed in Node.js 12
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)